### PR TITLE
Two alert dialogs issue solved

### LINKED ISSUE #37: Two alert dialogs appearing if no network connection
On starting app without network connection, two alert dialogs are created saying `No Internet Connection`.

### PR DESCRIPTION
**Pull request by mejariamol**
fixes #37

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export class OrganizerAppComponent {
 
   @ViewChild(Nav) public nav: Nav;
   public rootPage: any;
+  private disconnectSubscription: any;
```
Comment (niranjan94) [inline src/app/app.component.ts]: Since we know the type, don't use `any`

File: src/app/app.component.ts [hunk 2]
```diff
@@ -50,10 +51,15 @@ export class OrganizerAppComponent {
   }
 
   private handleNetworkChanges() {
-    Network.onDisconnect().subscribe(() => {
+    this.disconnectSubscription = Network.onDisconnect().subscribe(() => {
```
Comment (niranjan94) [inline src/app/app.component.ts]: We have code-duplication here. Please avoid duplication of code.

File: src/app/app.component.ts (hunk 1 above)
Comment (mejariamol) [inline src/app/app.component.ts]: I imported `Subscription` using `import {Subscription} from '@reactivex/rxjs/es6/Subscription.js'`.
Still giving error `Cannot find name 'Subscription'`.
Is there anywhere else where I will need to import the package??

Comment (niranjan94) [inline src/app/app.component.ts]: @mejariamol just a simple `import {Subscription} from "rxjs";` should do ...

Comment (mejariamol) [inline src/app/app.component.ts]: using `import {Subscription} from "rxjs";` 
giving error ` Type 'Subscription' is not assignable to type 'Subscription'. Property '_parent' is missing in type
            'Subscription'.`
what can be the problem?

Comment (niranjan94) [inline src/app/app.component.ts]: Do a `typeof` and see what is the actual type of `disconnectSubscription`

Comment (mejariamol) [inline src/app/app.component.ts]: it is returning `object`

Comment (niranjan94) [inline src/app/app.component.ts]: Weird. :thinking: Okay .. use `any` for now .. I'll check this out tomorrow

File: src/app/app.component.ts (hunk 2 above)
Comment (niranjan94) [inline src/app/app.component.ts]: Fix this one. And it's ready to merge

**Comment by mejariamol** [comment]
@niranjan94 The function `initDisconnectSubscription()` itself unsubscribes the subscription once error is shown. Hence there won't be any case of subscribing to `onDisconnect` twice.

**Comment by niranjan94** [comment]
@mejariamol one PR per issue please. Don't mix code/

**Comment by mejariamol** [comment]
@niranjan94 Shall I remove last commit?

**Comment by niranjan94** [comment]
Yes @mejariamol ... Remove and open a new PR

**Comment by mejariamol** [comment]
@niranjan94 Removed last commit. To make new PR for #39 this will need to be merged before right?

**Comment by niranjan94** [comment]
@mejariamol I don't see why this should be merged before making another PR ... You can always create a new branch and make a new PR

**Pull request merged**